analysis/code/analyze_data.py

```diff
@@ -31,7 +38,8 @@ def format_model(fit):
                               'p_value'  : fit.pvalues})
     formatted = formatted.loc[['post_tv']]
     
-    return(formatted)
+    return(formatted)  
+       
     
 ### EXECUTE
 main()
```

Comment:
Whenever you see the red icon above, it means `GitHub` is expecting a new line at the end of the file. You can resolve this by adding a new blank line.